webextensions/javascript-apis.json

```diff
@@ -3453,7 +3453,7 @@
                     "version_added": "48.0"
                   },
                   "opera": {
-                    "version_added": false
+                    "version_added": true
```

Comment:
I just tried out, with [one of my own extensions](https://github.com/snoack/mypass/tree/master/extension) which uses `tabs.executeScript`, on Opera 45 and it works fine. Without, `InjectDetails` there would be nothing you can do with `tabs.executeScript` or `tabs.insertCSS`.

BTW, how about removing `InjectDetails` here and from the MDN wiki? It seems on MDN the respective properties are documented redundantly for [executeScript](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/tabs/executeScript) and [insertCSS](https://developer.mozilla.org/en-US/Add-ons/WebExtensions/API/tabs/insertCSS) anyway. In fact, they aren't even the same anymore since `cssOrigin` has been added.